build-system/release-tagger/create-release.js

```diff
@@ -23,20 +23,113 @@
  */
 
 const argv = require('minimist')(process.argv.slice(2));
+const dedent = require('dedent');
 const {
   createRelease,
   getPullRequestsBetweenCommits,
   getRelease,
 } = require('./utils');
 const {GraphQlQueryResponseData} = require('@octokit/graphql'); //eslint-disable-line no-unused-vars
 
+/**
+ * Format pull request line
+ * @param {GraphQlQueryResponseData} pr
+ * @return {string}
+ */
+function _formatPullRequestLine(pr) {
+  const {mergeCommit, number, title, url} = pr;
+  const {abbreviatedOid, commitUrl} = mergeCommit;
+  return dedent`\
```

Comment:
It still weirds me out that this is JavaScript 😱

Reply:
Would it help readability if `dedent()` were used like a function? (Dev docs say it should work.)

Reply:
That is what the docs say, doesn't seem to work however.